dashboard/src/components/InfoCard/InfoCard.test.tsx

```diff
@@ -64,3 +64,19 @@ it("should parse JSX elements in the tags", () => {
   expect(tag2Elem).toExist();
   expect(tag2Elem.text()).toBe("tag2");
 });
+
+it("should trim a long tag", () => {
```

Comment:
If we don't ensure this the tag surpasses the card limit:

<img width="492" alt="screenshot 2018-12-04 at 18 24 57" src="https://user-images.githubusercontent.com/4025665/49462565-a9590880-f7f6-11e8-96bb-881dcdc2391b.png">

With the fix:

<img width="330" alt="screenshot 2018-12-04 at 18 23 58" src="https://user-images.githubusercontent.com/4025665/49462581-b544ca80-f7f6-11e8-84f3-096590941bc9.png">